components/usage/pkg/scheduler/scheduler.go

```diff
@@ -70,7 +70,7 @@ func (c *Scheduler) Start() {
 				return jobErr
 			})
 			if err != nil {
-				if errors.Is(err, redsync.ErrFailed) {
+				if errors.Is(err, redsync.ErrTaken{}) {
```

Comment:
Let's downgrade this case to Info as well, and then have a general handler with Error for the case where not ErrTaken

Reply:
I don't understand that comment. Can you explain?

Reply:
Ah, sorry. It's already info. I got confused with the `WithError` part. Ignore.